taichi/transforms/lower_matrix_ptr.cpp

```diff
@@ -97,6 +97,11 @@ class RemoveMatrixOfPtr : public BasicStmtVisitor {
     modifier_.erase(stmt);
   }
 
+  void visit(ArgLoadStmt *stmt) override {
```

Comment:
Hmm.. should we move it elsewhere? `RemoveMatrixOfPtr` is for cleaning redundant stmts after `LowerMatrixPtr`.